File: iGardenConnectAPI/VM/SmallGardenVM.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VM
{
    public class SmallGardenVM
    {
        public string IdGarden { get; set; }
        public string Name { get; set; }
        public short? Watered { get; set; }
        public DateTime? LastWatered { get; set; }
        public int? WateringDuration { get; set; }

        public PlantVM Plant { get; set; }

    }
}

File: igardenconnectweb/model/SmallGardenVM.ts
import { PlantVM } from "./PlantVM";

export class SmallGardenVM {
  idGarden?: string;
  name?: string;
  watered?: number;
  lastWatered?: Date;
  wateringDuration?: number;
  plant?: PlantVM;

  constructor(
    idGarden?: string,
    name?: string,
    watered?: number,
    lastWatered?: Date,
    wateringDuration?: number,
    plant?: PlantVM
  ) {
    this.idGarden = idGarden;
    this.name = name;
    this.watered = watered;
    this.lastWatered = lastWatered;
    this.wateringDuration = wateringDuration;
    this.plant = plant;
  }
}
